aiohttp/web.py

```diff
@@ -269,7 +269,7 @@
                    backlog: int=128,
                    access_log_class: Type[AbstractAccessLogger]=AccessLogger,
                    access_log_format: str=AccessLogger.LOG_FORMAT,
-                   access_log: logging.Logger=access_logger,
+                   access_log: Optional[logging.Logger]=access_logger,
```

Comment:
why do u use `Optional`, and when access_log should be None?

Reply:
i just don't understand when u need to drop logging :)

Reply:
Firstly, this is just an adoption of the stubs to reflect existing [docs](https://docs.aiohttp.org/en/stable/logging.html#access-logs), secondly I ask myself the vice versa question every time I try to get rid of some framework's logging. I have “dumb” HTTP logs in my load balancer and I want my applications to have only structured application logs.

Reply:
The load balancer is story about production, right? do u use debug mode for production?
Now, u can put your custom logger and this should solve your problem.

Reply:
This isn't about *my* problems, this is about making the code do what the docs say. :)

Reply:
so maybe the problem is not in the code and in the documentation :)
I don't find any test for this behavior and this feature never worked. I am not sure that this feature is required.

Reply:
It worked when debug was off which is the normal case. I ran into the edge case of debug=True thanks to mypy.

Reply:
You are right without mypy all is fine. I think will be cool have some test for this case.